businesscentral-indexing/modules/businesscentral-indexing/module.yaml

```diff
@@ -0,0 +1,5 @@
+schema_version: 1
+
+name: businesscentral-indexing
```

Comment:
why not use os.businesscentral.indexing?

Reply:
or os.bpmsuite.indexing

Reply:
This follows the same naming convention of all the other standalone modules. It's the openshift modules in jboss-kie-modules that use the dot notation of os.bpmsuite.name.